README.md

```diff
@@ -1,9 +1,5 @@
 <p align="center">
   <h1 align="center"><a href="https://codeforaustralia.github.io/seechange/">SeeChange (aka. Civic Makers Climate-Change) Project</a></h1>
```

Comment:
Hi @klubiniecki ,

Thanks for the PR! Apologies for the tardy review! :bow: 

The changes look awesome :+1: 

One minor addition/suggestion: Can we change the url to website from `https://codeforaustralia.github.io/seechange/` to `https://seechange.codeforaustralia.org/` ?